test/asf-gdal/GDAL_Subsetter_Regression.ipynb

```diff
@@ -57,16 +57,10 @@
    },
    "outputs": [],
    "source": [
-    "SUPPORTED_ENVS = ['uat']\n",
-    "global env\n",
-    "m = re.search('harmony(\\.|-)(.*)(\\.earthdata\\.nasa\\.gov|-frontend)', harmony_host_url)\n",
-    "env = m.group(2)\n",
-    "if env.lower() not in SUPPORTED_ENVS:\n",
-    "    print(f'{env} is not a supported environment (supported envs: {SUPPORTED_ENVS}) - skipping tests')\n",
-    "    sys.exit(0)\n",
-    "print(\"Environment: \", env)\n",
-    "if env.lower() == 'sit':\n",
-    "    print('Connect to the NASA VPN before running requests against the sit environment.')"
+    "SUPPORTED_ENVS = ['harmony.uat.earthdata.nasa.gov']\n",
+    "if harmony_host_url != 'https://harmony.uat.earthdata.nasa.gov':\n",
```

Comment:
This also means the notebook won't work for sandbox, SIT, or local, which all tend to point at UAT.  Is that the desired behavior?

Reply:
Good point about them pointing to UAT. The key is limiting where the access is coming from, so I will test this notebook in those environments. This restriction was originally added by ASF (modified by me).

Reply:
Wait - no. We only want to run in UAT, because on UAT can access the data. Not an issue with accessing CMR UAT.